src/spec.c

```diff
@@ -1959,11 +1996,18 @@ static int FieldSpec_RdbLoad(RedisModuleIO *rdb, FieldSpec *f, StrongRef sp_ref,
     return FieldSpec_RdbLoadCompat8(rdb, f, encver);
   }
 
-  LoadStringBufferAlloc_IOErrors(rdb, f->name, NULL, goto fail);
-  f->path = f->name;
+  char* name = NULL;
+  size_t len = 0;
+  LoadStringBufferAlloc_IOErrors(rdb, name, &len, true, goto fail);
+  if (!len)
+    goto fail;
```

Comment:
Can we cover this? It's a delicate place.

Reply:
Looking at the rlookup branch I see it was removed, I think there was a test where the rdb we loaded from contained an empty string with length 1.
Will remove the if.